data/example_vocab.csv

```diff
@@ -0,0 +1,1001 @@
+word,logfreq,wlen
+rematch,-12.87097864,7

```

Comment:
this file should be uploaded by the user. will add to readme next time.

Reply:
so do you mean you're removing it from the repo? It seems like it would be useful to include something, even if the user can replace it themselves. And if we include something, we should explain where it comes from.